packages/gatsby/src/redux/actions/public.js

```diff
@@ -1158,4 +1158,35 @@ actions.createRedirect = ({
   }
 }
 
+/**
+ * Create a dependency between a page and data.
+ *
+ * @param {Object} $0
+ * @param {string} $0.path the path to the page
+ * @param {string} $0.nodeId A node ID
+ * @param {string} $0.connection A connection type
+ * @private
+ */
+actions.createPageDependency = (
+  {
+    path,
+    nodeId,
+    connection,
+  }: { path: string, nodeId: string, connection: string },
+  plugin: string = ``
+) => {
+  console.warn(
+    `Calling "createPageDependency" directly from actions in deprecated. Use "createPageDependency" from "gatsby/dist/redux/actions/add-page-dependency".`
```

Comment:
I don't quite understand the warning message - people should import it from gatsby internals?

Reply:
It is described in docs https://www.gatsbyjs.org/docs/page-node-dependencies/#add-page-dependencyjs